tools/config/__init__.py

```diff
@@ -1354,10 +1354,10 @@ def get_features(self):
         self.cumulative_overrides['features']\
             .update_target(self.target)
 
-        for feature in self.target.features:
+        for feature in list(self.target.features):
```

Comment:
You might consider adding a comment why this needs to be a list (because we are modifying the features list and need to iterate on a copy).